### ISSUE TITLE
ImportError: cannot import name 'Pointclouds'

### ISSUE THREAD

**Issue by zhangyulong882** [facebookresearch/pytorch3d#160]
I  installed pytorch3d use the command below:
 pip install pytorch3d

and when I want to import Pointclouds from pytorch3d.structures package, I got the error:
ImportError: cannot import name 'Pointclouds'

My Code is:
from pytorch3d.structures import Pointclouds

**Comment by bottler** [comment]
You have installed the latest released version of pytorch3d, which is version 0.1.1. Pointclouds is something we have added recently on github, which has not been released yet. We hope to release a new version, which will include Pointclouds soon. 

If you can compile extensions (which is likely) you can install straight from github with 
```
pip uninstall pytorch3d
pip install 'git+https://github.com/facebookresearch/pytorch3d.git'
```

Can you update or close your other issue please?

**Comment by zhangyulong882** [comment]
It worked. Thanks
